docker-compose/docker-compose.yml

```diff
@@ -40,7 +40,7 @@ services:
   # WHISK CONTROLLER
   controller:
     image: ${DOCKER_IMAGE_PREFIX:-openwhisk}/controller
-    command: /bin/sh -c "exec /init.sh 0 >> /home/owuser/controller-local_logs.log 2>&1"
+    command: /bin/sh -c "exec /init.sh 0 >> /logs/controller-local_logs.log 2>&1"
```

Comment:
does the process have access to write in `/logs` actually ?  Alternatively we can do https://github.com/apache/incubator-openwhisk-devtools/pull/259/commits/b9753eb30239345fe6e2f64e6d3e8e2b70025618#diff-cb44929492b826b5b16656db6a5a26a4R76

Reply:
Yes, I think that the process has permissions to write under `/logs` as, at least in my deployment from docker-compose, is running as root.

Redirecting them to `/logs` is consistent with the invoker logs being there as well (as in https://github.com/apache/incubator-openwhisk-devtools/pull/263/commits/9cf4541dc937bafb1ee7246b7a5e8aa342fd4436#diff-cb44929492b826b5b16656db6a5a26a4R85). I'd rather redirect logs inside the container to the same location, being it `/logs` or `/home/owuser`.